x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/schema.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { MakeSchemaFrom } from '@kbn/usage-collection-plugin/server';
+import type { CspmUsage } from './types';
+
+export const cspmUsageSchema: MakeSchemaFrom<CspmUsage> = {
+  indices: {
```

Comment:
💯 
Had you check that null values doesn't break our visualizations?
If it does break anything - suggesting to add another boolean or another field for the index state

Reply:
I test with null timestamp and it worked well